parts/linux/cloud-init/artifacts/components.json

```diff
@@ -443,7 +443,7 @@
         "ubuntu": {
```

Comment:
what about azurelinux?

Reply:
I believe azure linux would fall under the "default" case, is that right @Devinwong ?

Reply:
yes, if there is "mariner", it will use the mariner one. Otherwise it will use the default one

Reply:
so can we remove the `ubuntu-->current` is this should be the same as the default ? I"m guessing the property `ubuntu` is mainly for `r1804` ?